inventory_disbursements/models.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from items.models import Item
+
+
+class Disbursement(models.Model):
```

Comment:
For the on_delete=models.Cascade, not sure we want to cascade the changes because for instance, if an item is deleted, we still want to see that the item was disbursed some time ago. Maybe we can do something similar to what we did for requests.